packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,130 +3,184 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-description-color: Specifies the color of the component's description.
+ * @prop --calcite-panel-fab-z-index: Specifies the component fab's z-index.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
  * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
+ * @prop --calcite-panel-header-background-color: Specifies the component header's background color.
  * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+ * @prop --calcite-panel-header-color: Specifies the component header's  color.
```

Comment:
Should this be `heading` to correspond to the property name and match `description-`?

Reply:
I'm torn on this one. It does set the color for the header but I could see your point. What if there is more text in the header in the future? Would it affect slotted actions?

Reply:
I suppose both could be valid as well - the `header` here could apply to any slotted `header-content`, and a `heading` one could apply just to the rendered `heading` prop?

Reply:
Can we just add the heading one later if we need to? @macandcheese

Reply:
Yeah that’s fine. I guess between the two I’d expect ‘heading’ since we render that text for a user based on prop and slotted ‘header’ content is already user-controllable.

Reply:
I don’t think we’d want it to affect slotted actions though, since those are individually theme able already. Would it just be the rendered overflow “ellipsis” action affected by this? Besides slotted content?

Reply:
> Would it just be the rendered overflow “ellipsis” action affected by this? Besides slotted content?

I think so.

Reply:
I guess my feeling is `heading` is more useful here for the "I want to change the text color of heading and description" theming need. But, ultimately we can add more later if needed.

re: the border ones:
Can / should the other one be simplified to `--calcite-panel-header-border-color` ? The block-end seems a little in the weeds for the user. This also makes me think there would be a "footer border" prop too.

Maybe we don't even need to allow that to be overridden - trying to think if we'd ever want that to be themed separately from the panel border itself (maybe set to transparent for some reason?)

Reply:
We could deprecate it and leave it until the next breaking change release if we want to I'd think. But, yeah seems a little strangely named.

Reply:
I agree it's strangely named. I'd vote to deprecate if we can.

Reply:
- [ ] deprecate --calcite-panel-header-color